04-understanding-survey-data-documentation.Rmd

```diff
@@ -104,7 +106,7 @@ The survey documentation, often the codebook, represents the missing data with a
 knitr::include_graphics(path="images/codebook-example.jpg")
 ```
 
-When running analysis in R, we must missing responses as missing data (i.e., `NA`) and not numeric data. If missing responses are treated as zeros or arbitrary values, they can artificially alter summary statistics or introduce spurious patterns in the analysis.
+When running analysis in R, we must missing responses as missing data (i.e., `NA`) and not numeric data. If missing responses are treated as zeros or arbitrary values, they can artificially alter summary statistics or introduce spurious patterns in the analysis. 
```

Comment:
Wording isn't quite right in first sentence. Perhaps ...we must **set** missing responses as missing data...